content/en/docs/concepts/signals/metrics.md

```diff
@@ -73,6 +73,33 @@ The instrument kind is one of the following:
   A histogram is a good choice if you are interested in value statistics. For
   example: How many requests take fewer than 1s?
 
+### Synchronous and asynchronous instruments
+
```

Comment:
I miss an introductory sentence here that presents the two possible types of instruments. The order should also be the same as the title, and so start with synchronous and then follow with async.